controller/commentsController.js

```diff
@@ -47,3 +48,42 @@ exports.flagComment = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.updateComment = async (req, res, next) => {
+  const comment_id = req.params.commentId;
+  const commentBody = req.body.content;
+  const owner = req.body.ownerId;
+
+  Comments.findById(comment_id)
+    .exec()
+    .then((comment) => {
+      if (!comment) {
+        return new CustomError(404, "Comment not found");
+      } else if (comment.ownerId != owner) {
+        return new CustomError(
+          403,
+          "Sorry, comment cannot be updated or Unauthorized"
+        );
+      }
+      Comments.updateOne(
+        { _id: comment_id },
+        { $set: { commentBody: commentBody, isEdited: true } }
```

Comment:
There is no commentBody anymore, it's been replaced with content so please update accordingly.